Locker.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Locker"
-  s.version      = "3.0.4"
+  s.version      = "3.0.5"
```

Comment:
@nsimunko, please return this to 3.0.4 as the currently released version is 3.0.3, so the next one should be 3.0.4.